public/app/percona/dbaas/components/DBCluster/__mocks__/DBCluster.service.ts

```diff
@@ -9,4 +10,7 @@ export class DBClusterService {
   static async getAllocatedResources(): Promise<DBClusterAllocatedResources> {
     return Promise.resolve(dbCLusterAllocatedResourcesStub);
   }
+  static async getDBClusters() {
+    return Promise.resolve();
```

Comment:
Should this have a default value? Or is undefined fine in this case?

Reply:
In the most cases we have different values of this mock, so i didn't add one value there and i think it's okey, because in other cases we didn't need value, just fact of resolve